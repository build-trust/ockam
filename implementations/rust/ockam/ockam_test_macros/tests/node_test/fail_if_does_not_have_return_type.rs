#[ockam_test_macros::node_test]
async fn my_test(ctx: &mut ockam_node::Context) {
    ctx.address();
}
