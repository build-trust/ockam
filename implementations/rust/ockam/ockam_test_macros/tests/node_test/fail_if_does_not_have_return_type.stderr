error: The function must have a return type
 --> $DIR/fail_if_does_not_have_return_type.rs:2:7
  |
2 | async fn my_test(ctx: &mut ockam_node::Context) {
  |       ^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> $DIR/fail_if_does_not_have_return_type.rs:1:1
  |
1 | / #[ockam_test_macros::node_test]
2 | | async fn my_test(ctx: &mut ockam_node::Context) {
3 | |     ctx.address();
4 | | }
  | |_^ consider adding a `main` function to `$DIR/tests/node_test/fail_if_does_not_have_return_type.rs`
